keyboards/pica40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 zzeneg (@zzeneg)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "quantum.h"
```

Comment:
Shouldn't be needed. 
```suggestion
```

Reply:
It doesn't compile without it actually, throwing errors like `error: unknown type name 'uint16_t'`. Maybe something else is missing?

Reply:
moved that include to rev*.h files